straxen/get_corrections.py

```diff
@@ -31,9 +31,17 @@ def get_to_pe(run_id, gain_model, n_tpc_pmts):
             raise ValueError('CMT gain model should be similar to:'
                              '("CMT_model", ("to_pe_model", "v1"). Instead got:'
                              f'{model_conf}')
-        is_nt = n_tpc_pmts == straxen.n_tpc_pmts
+        # is this the best way to do this?
+        is_nt = n_pmts == straxen.n_tpc_pmts or n_pmts == straxen.n_nveto_pmts or n_pmts == straxen.n_mveto_pmts
```

Comment:
I think it's intuitive enough. For readability you might want to do something like:

```python
is_nt = n_pmts in [straxen.n_tpc_pmts, straxen.n_nveto_pmts, straxen.n_mveto_pmts]
```

Reply:
Also this does not work, at least not for the muon-veto. In case of the muon-veto it is the same number of PMTs in 1T and nT.

Reply:
Sorry Daniel but I don't follow. This is where you get the argument that you need to be able to do the init in the first place.

>  In case of the muon-veto it is the same number of PMTs in 1T and nT.
True, but there is no such thing at present as `strax xenon1T MV data` . I'm not aware of any plans of porting this data to the strax format but perhaps you know more than I do?

Reply:
Daniel as Joran mentioned we need to do the init to look either nT or 1T runDB

Reply:
> Sorry Daniel but I don't follow. This is where you get the argument that you need to be able to do the init in the first place.

Yes, sorry I confused myself here. 

> True, but there is no such thing at present as strax xenon1T MV data . I'm not aware of any plans of porting this data to the strax format but perhaps you know more than I do?

I am not aware of this for the moment. But I think it is not unlikely that you will compare at some point Strax with PAX also for the muon-veto. 

> Daniel as Joran mentioned we need to do the init to look either nT or 1T runDB

True, I understand this, but I do not like this semi-correct solution here which works only because we think non will ever use it. I think also for other reasons it would be actually nicer if the initialization of CMT becomes part of the contexts. Maybe not needed to change it right away in this PR, but at least I think it is worth to think about it, is not it? 
As a fall back solution one could do it in a similar way as in the electron life time.

Reply:
I see your point. Given that there is neither 1T gains nor 1T strax data for the MV I would not worry about it too much at this stage.

You raise a valid point though that we could think of something that keeps the CMT initialized for the right detector at ~ the context level.